source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.IntegrationTests/DomainTests/ChargeLinkIngestionTests.cs

```diff
@@ -52,18 +53,18 @@ public Task DisposeAsync()
             [Fact]
             public async Task When_ChargeIsReceived_Then_AHttp200ResponseIsReturned()
             {
-                var request = CreateHttpRequest("CreateFixedPeriodTariffChargeLink.xml", out _);
+                var request = CreateHttpRequest(ChargeLinkDocument.AnyValid, out _);
 
                 var actualResponse = await Fixture.HostManager.HttpClient.SendAsync(request);
 
                 actualResponse.StatusCode.Should().Be(HttpStatusCode.OK);
             }
 
             [Fact]
-            public async Task When_ReceivingChargeLinkMessage_MessageHubIsNotifiedAboutAvailableData_And_Then_When_MessageHubRequestsTheBundle_Then_MessageHubReceivesBundleReply()
+            public async Task Given_NewChargeLinkMessage_When_GridAccessProviderPeeks_Then_MessageHubReceivesReply()
```

Comment:
As the Grid Access Provider only gets notified when the linked charge is a tax, perhaps we should make this explicit by renaming the test to: 
```suggestion
            public async Task Given_NewTaxChargeLinkMessage_When_GridAccessProviderPeeks_Then_MessageHubReceivesReply()
```

Reply:
Great idea. Fixed in [this commit](https://github.com/Energinet-DataHub/geh-charges/pull/857/commits/8588cea21175525b35b055be7d473e60a6061e80).